package.json

```diff
@@ -36,7 +36,7 @@
   "dependencies": {
     "@edx/brand": "npm:@openedx/brand-openedx@^1.2.2",
     "@edx/frontend-component-header": "5.3.3",
-    "@edx/frontend-platform": "7.1.4",
+    "@edx/frontend-platform": "8.0.0",
```

Comment:
The latest version of platform is `8.0.4` - is there a specific reason to keep this pinned to an exact version? It seems just using `^8.0.0` might be a reasonable option here.